### PR TITLE
Added some NPC lines, tweaked some.

### PR DESCRIPTION
**Pull request by onpon4**

### CODE REVIEW

**Comment by onpon4** [comment]
Pushed a fix to the FLF Catastrophe event here as well (it was attempting to see if a no longer existent mission was active, breaking it).

File: dat/events/flf/flf_catastrophe.lua [hunk 1]
```diff
@@ -58,10 +58,10 @@ title[11] = _("Welcome To the Nebula")
 text[11] = _([[As you land on the mysterious station, armed guards immediately surround your ship and order you out into the hangar. You comply, and they take you to a room that appears to be an interrogation room, where you wait for a few nerve-wracking hectoseconds. Finally, a holoscreen flickers on, showing the figure you had seen earlier. She looks in your direction.
     "Well met," she says. "I see you have made the right choice and survived the nebula. Very good."]])
 
-text[12] = _([["I'm sure you're wondering who we are. We are the Thurion, a superior people and one of the Empire's former secret projects. Have you met the Za'lek? The Collective? Those too were the Empire's "great projects", as were the now dead Proteron. But we were kept a lot more in the dark.
+text[12] = _([["I'm sure you're wondering who we are. We are the Thurion, a civilization leftt over from one of the Empire's former secret projects. Have you met the Za'lek? The Collective? Those too were the Empire's "great projects", as were the now dead Proteron. But we were kept a lot more in the dark.
```
Comment (bobbens) [inline dat/events/flf/flf_catastrophe.lua:61]: leftt has two ts

**Comment by onpon4** [comment]
Fixed that typo

**Comment by bobbens** [comment]
Looks good, thanks!

**Pull request merged**